editor/Builder.cs

```diff
@@ -129,8 +120,7 @@ private static void addFile(ZipArchive archive, string path, string sourceDirect
             Trace.WriteLine($"  Adding {path} -> {entryName}");
             archive.CreateEntryFromFile(path, entryName, CompressionLevel.Optimal);
 
-            var pathExtension = Path.GetExtension(path);
-            if (pathExtension == ".exe" || pathExtension == ".dll")
+            if (Path.GetExtension(path) == ".dll")
```

Comment:
In .NET <= 4.X, all code is contained in the exe file. From .NET 5 on, all code is contained in a dll file (StorybrewEditor.dll), and the exe file only contains a bootloader for the CLR (StorybrewEditor.exe). Including the exe file too results in two `StoryboardEditor.pdb` entries in the zip archive (yes thats possible) and when unpacking it fails because it tries to write a file that already exists.